AnkiDroid/src/main/java/com/ichi2/libanki/Decks.java

```diff
@@ -520,17 +520,15 @@ private boolean _canDragAndDrop(String draggedDeckName, String ontoDeckName) {
 
 
     private boolean _isParent(String parentDeckName, String childDeckName) {
-        List<String> parentDeckPath = path(parentDeckName);
-        List<String> childDeckPath = path(childDeckName);
+        String[] parentDeckPath = path(parentDeckName);
+        String[] childDeckPath = path(childDeckName);
         
-        if (parentDeckPath.size() +1 != childDeckName.size()) {
+        if (parentDeckPath.length +1 != childDeckPath.length) {
```

Comment:
whitespace around operators so they stand out

```suggestion
        if (parentDeckPath.length + 1 != childDeckPath.length) {
```